lib/app/store.js

```diff
@@ -11,12 +11,8 @@ const filenames = files.keys()
 let storeData = {}
```

Comment:
Y'all mind if I put a PR here in the comments?
No? Ok, good:
```js
import Vue from 'vue'
import Vuex from 'vuex'

Vue.use(Vuex)

// Recursive find files in {srcDir}/{dir.store}
const files = require.context('@/<%= dir.store %>', true, /^\.\/(?!<%= ignorePrefix %>)[^.]+\.(<%= extensions %>)$/)
const filenames = files.keys()

// Check if {dir.store}/index.js exists
const indexFilename = filenames.find(filename => filename.includes('./index.'))


// Store
const storeData = indexFilename ? getModule(indexFilename) : {}

const PREFIX_EXT_EX = /^\.\/|\.(<%= extensions %>)$/g

// If store is not an exported method = modules store
if (typeof storeData !== 'function') {
  // Store modules
  if (!storeData.modules) {
    storeData.modules = {}
  }

  for (const filename of filenames) {
    const cleanName = filename.replace(PREFIX_EXT_EX, '')
    if (cleanName === 'index') continue

    const namePath = cleanName.split("/")

    const basename = namePath[namePath.length - 1]
    if (['state', 'getters', 'actions', 'mutations'].includes(basename)) {
      const module = getModuleNamespace(storeData, namePath, true)
      appendModule(module, filename, basename)
      continue
    }

    // if file is foo/index.js
    // it should save as foo
    const isIndex = (basename === 'index')
    if (isIndex) {
      namePath.pop()
    }

    const module = getModuleNamespace(storeData, namePath)
    const fileModule = getModule(filename)
    const nonIndexBasename = namePath.pop()
    let newModule = module[nonIndexBasename] || {}

    const assignPriority = [{}]
    if (!isIndex) {
      // if file is foo.js, existing properties take priority
      // because it's the least specific case
      assignPriority.push(fileModule, newModule)
    } else {
      // if file is foo/index.js we want to overwrite properties from foo.js
      // but not from appended mods like foo/actions.js
      assignPriority.push(newModule, fileModule)
      if (newModule.appends) {
        const appendedMods = {}
        appendedMods.appends = newModule.appends
        for (const append of newModule.appends) {
          appendedMods[append] = newModule[append]
        }
        assignPriority.push(appendedMods)
      }
    }
    assignPriority.push({namespaced: true})
    module[nonIndexBasename] = Object.assign.apply(undefined, assignPriority)
  }
}

// createStore
export const createStore = storeData instanceof Function ? storeData : () => new Vuex.Store(Object.assign({
    strict: (process.env.NODE_ENV !== 'production')
}, storeData, {
        state: storeData.state instanceof Function ? storeData.state() : {}
    }))


// Dynamically require module
function getModule (filename) {
  const file = files(filename)
  const module = file.default || file
  if (module.commit) {
    throw new Error('[nuxt] <%= dir.store %>/' + filename.replace('./', '') + ' should export a method which returns a Vuex instance.')
  }
  if (module.state && typeof module.state !== 'function') {
    throw new Error('[nuxt] state should be a function in <%= dir.store %>/' + filename.replace('./', ''))
  }
  return module
}

function getModuleNamespace (storeData, namePath, forAppend = false) {
  if (namePath.length === 1) {
    if (forAppend) { return storeData }
    return storeData.modules
  }
  const namespace = namePath.shift()
  const newModule = storeData.modules[namespace] || { modules: {} }
  newModule.modules || (newModule.modules = {})
  newModule.namespaced = true
  storeData.modules[namespace] = newModule
  return getModuleNamespace(newModule, namePath, forAppend)
}

function appendModule (module, filename, name) {
  const file = files(filename)
  module.appends || (module.appends = [])
  module.appends.push(name)
  module[name] = file.default || file
}
```

Reply:
I do mind ;)

Reply:
If you really want to contribute instead of trolling, open up your own PR.

Reply:
It's bound to conflict with this one, and I am on  a  trip 🤔